src/helpers/categoryHelper.ts

```diff
@@ -47,3 +49,21 @@ export function getSelectedGeography(pageUrl) {
     return { geoType: "ew", geoCode: "K04000001" };
   }
 }
+
+export const renderCatHeadString = (
```

Comment:
Ditto on the naming - consider `formatCategoryHeader` - but I would move this personally inside the component - it's completely coupled / belongs to this little component.

Reply:
done - for readers, conversation offline about above - 'util' is for very generic code, 'helper' is for code that could be used across multiple components / places in the app, in-component for component-specific logic (rules of thumb)